docs/src/tutorials/tutorial_2.md

```diff
@@ -1,5 +1,367 @@
 # **Tutorial 2:** What Are Actions?
 
+In this tutorial, we are going to learn how to make a brain! 🧠 
+Well, not _exactly_ making a brain. 
+Instead, we are going to animate brain activity by simulating a [10-20 EEG Electrode Array](https://en.wikipedia.org/wiki/10%E2%80%9320_system_(EEG)?oldformat=true) using random data. 
+This Project Tutorial builds on prior tutorials and serves as a cumulative test of your understanding about `Javis`.
 
+When you are done with this tutorial, you will have created the following animation:
 
+![](assets/eeg.gif)
+
+# Learning Outcomes
```

Comment:
Maybe use `##`  here and for the following subsections. It should be smaller than the `# **Tutorial 2:** What Are Actions?` heading